nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -720,11 +741,12 @@ public Document copy(Document source, Document parent, String name) {
         return getDocument(copyState);
     }
 
-    protected String copyRecurse(String sourceId, String parentId, LinkedList<String> ancestorIds, String name) {
+    protected String copyRecurse(String sourceId, String parentId, LinkedList<String> ancestorIds, String name, boolean excludeSpecialChildren) {
         String copyId = copy(sourceId, parentId, ancestorIds, name);
         ancestorIds.addLast(copyId);
-        for (String childId : getChildrenIds(sourceId)) {
-            copyRecurse(childId, copyId, ancestorIds, null);
+        for (String childId : getChildrenIds(sourceId, excludeSpecialChildren)) {
+            // the special flag is not passed recursively
```

Comment:
-> `// don't exclude special children when recursing`

Reply:
I used this comment but I put regular instead of special:
The special children are just at the first children level, so we don't want to exclude regulars when recursing.
I copy commentRoot which is special, then I want to copy the regulars inside so I put the hardcoded flag like in VCS.